packages/http/src/forwarder/index.ts

```diff
@@ -41,9 +41,11 @@ const forward: IPrismComponents<IHttpOperation, IHttpRequest, IHttpResponse, IHt
       TE.chainEitherK(() => serializeBody(input.body)),
       TE.chain(body =>
         TE.tryCatch(async () => {
-          const partialUrl = parse(baseUrl);
+          const partialUrl = new URL(baseUrl);
           const url = format({
             ...partialUrl,
+            host: partialUrl.host,
```

Comment:
why include this if is already spread in from line 46?

Reply:
Since `partialUrl.host` is a getter not property. It is not copied properly. 

Come to think of it, it would be better to use `Object.assign` instead of spread. :)